rpcs3/Emu/RSX/GCM.cpp

```diff
@@ -763,6 +763,33 @@ rsx::primitive_type rsx::to_primitive_type(u8 in)
 }
 
 
+#define CELL_GCM_WINDOW_ORIGIN_TOP  0
```

Comment:
Defines should be avoided according to our style guide. Why remove them from the enum?